### ISSUE TITLE
Get Uncaught TypeError when delete a note in the panel

### ISSUE THREAD

**Issue by htyeim** [jplattel/open-note-to-window-title#19]
This is not a big problem and will not cause any crush. So far, the only effect is errors shown in the Console (`alt+cmd+i`) as below:
```js
VM267:133 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'path')
    at ActiveNoteTitlePlugin.refreshTitle (eval at <anonymous> (app.js:1:1), <anonymous>:133:68)
    at ActiveNoteTitlePlugin.eval (eval at <anonymous> (app.js:1:1), <anonymous>:46:18)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app.js:1:1), <anonymous>:26:71)
    at new Promise (<anonymous>)
    at __awaiter (eval at <anonymous> (app.js:1:1), <anonymous>:22:12)
    at ActiveNoteTitlePlugin.handleDelete (eval at <anonymous> (app.js:1:1), <anonymous>:45:39)
    at t.e.tryTrigger (app.js:1:903075)
    at t.e.trigger (app.js:1:903008)
    at t.trigger (app.js:1:919684)
```
However, I usually open the console panel, and sometimes, I need to delete some temp notes, and the red error text is really annoying... I am wondering if it could be fixed...

the content of the related line is:
```js
template = Object.assign({ 'parentpath': (file.parent).path, 'filepath': file.path, 'filename': file.name, 'basename': friendlyBasename, 'extension': file.extension }, template);
```

**Comment by luckman212** [comment]
@htyeim I just pushed a [test commit](https://github.com/luckman212/open-note-to-window-title/commit/2a05d1736d9cb35447e2058401c2b75809b01bc0) that seems to fix this on my system. Would you mind testing it?

If you have a working nodejs setup, you can clone that branch with

```
git clone --branch try-to-fix-typeerror https://github.com/luckman212/open-note-to-window-title
```
and build it yourself with
```
npm run build
```
otherwise, you can download this precompiled [`main.js`](https://github.com/jplattel/open-note-to-window-title/files/8750137/main.js.zip), unzip & drop it in your `/.obsidian/plugins/open-note-to-window-title` folder, replacing the current one.

**Comment by luckman212** [comment]
@jplattel in case this fix does indeed work, I submitted PR #20

**Comment by htyeim** [comment]
Yes, the branch fix the problem. The console shows no more red error message. Thanks!!

PS, after cloning, I need `npm i` first, then `npm run build`  😄.
